client/verta/verta/environment/_environment.py

```diff
@@ -29,20 +29,6 @@ def __init__(self, env_vars, autocapture, apt_packages=None):
             self._capture_env_vars(env_vars)
         if autocapture:
             self._capture_cmd_line_args()
-        if apt_packages:
-            self.apt_packages = apt_packages
-
-    @property
-    def apt_packages(self):
-        return list(self._msg.apt.packages)
-
-    @apt_packages.setter
-    def apt_packages(self, packages):
-        if packages:
-            apt_blob = _EnvironmentService.AptEnvironmentBlob(packages=packages)
-            self._msg.apt.CopyFrom(apt_blob)
-        else:
-            self._msg.apt.Clear()
```

Comment:
apt packages don't really make sense for Docker environments (the Docker image should already have all the apt packages it needs) so I'm moving it the API for it out of the base class and to `Python`.